lib/elixir/lib/enum.ex

```diff
@@ -2494,9 +2499,17 @@ defmodule Enum do
   defp enum_to_string(entry) when is_binary(entry), do: entry
   defp enum_to_string(entry), do: String.Chars.to_string(entry)
 
-  defp random_index(n) do
-    :rand.uniform(n + 1) - 1
-  end
+  @spec random_integer(integer, integer) :: integer
+  defp random_integer(lower_limit \\ 0, upper_limit)
```

Comment:
I usually don't like using default arguments for private functions. And this one is making the first argument optional, which is even less clearer. What if we remove the default arg and pass it explicitly above? :)